app/views/admin/world_location_news/show.html.erb

```diff
@@ -1,24 +1,25 @@
-<% page_title @world_location_news.name %>
-<% page_class "world-location-show" %>
-
-<%= content_tag_for(:div, @world_location_news) do %>
-  <div class="world-location-header">
-    <h1>
-      <span class="name"><%= @world_location_news.name %></span>
-    </h1>
-    <p>
-    <%= link_to "View on website", @world_location_news.public_url(cachebust_url_options) %>
-  </div>
-
-  <section class="world-location-details">
-    <%= tab_navigation_for(@world_location_news) %>
-
-    <div class="mission-statement lead">
+<% content_for :page_title, @world_location_news.name %>
+<% content_for :context, "World location" %>
+<% content_for :title, @world_location_news.name %>
+<% content_for :title_margin_bottom, 4 %>
+<div class="govuk-!-margin-bottom-8">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <p class="govuk-body">
+        <%= link_to "View on website", @world_location_news.public_url(cachebust_url_options), class: "govuk-link" %>
+      </p>
+      <%= render "components/secondary_navigation", {
+        aria_label: "Document tabs",
```

Comment:
Is this the correct label?

Reply:
Not sure what you mean? If Aria_label Can we take this to a PR on the Guide and get best practice for Aria_labels